### ISSUE TITLE
Thread type

### ISSUE THREAD

**Issue by Laim** [zrh4/CrossStitchPatternMaker#1]
Hi! 

What thread type does this refer to? (DMC, Anchor etc)

**Comment by zrh4** [comment]
Thanks for asking. It's DMC floss colors.

**Comment by Laim** [comment]
Thanks @zrh4, noticed a few minutes after asking this! 

Another one if you don't mind, do you plan to add the option to choose the stitch per inch?  Just trying to work out at the moment how the size of the stitches are chosen?

Thanks for your quick reply too. <3

**Comment by zrh4** [comment]
I'm not very knowledgeable about cross-stitching, sorry. I had to defer to my wife.

From what she told me, the cloth you use determines the stitches per inch. The program right now just does 1 pixel = 1 stitch. I attached an example cross stitch made with the program using 18-count cloth.

Hopefully this clears things up? 😅 
![link cs](https://user-images.githubusercontent.com/14111745/74355289-92c41f00-4db4-11ea-8d2d-70f3a6cebaa9.jpg)
